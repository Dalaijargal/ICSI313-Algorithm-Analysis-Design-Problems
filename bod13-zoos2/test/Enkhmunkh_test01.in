919296 38
1091 1123 2083 2347 3167 4001 4621 5503 5647 6263 7219 7481 8087 8231 8831 9109 9419 9677 10103 10301 10657 10883 11161 11329 11617 11801 12071 12421 12763 13037 13109 13679 14143 14447 14621 14869 15139 15401