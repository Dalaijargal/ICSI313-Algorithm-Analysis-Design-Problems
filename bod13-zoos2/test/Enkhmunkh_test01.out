6461091607504867807
